tasks/wait-for-node-ready.yml

```
@@ -1,18 +1,18 @@
 ---
 # for master nodes wait for the master API port to be ready
 - block:
-  - name: OCP | Wait for Node Ready | Determine master API port
+  - name: OCP | Wait for Master Ready | Determine master API port
     set_fact:
       openshift_master_api_port: 443
     when: openshift_master_api_port is not defined
 
-  - name: OCP | Wait for Node Ready | Wait for master API to come back online
+  - name: OCP | Wait for Master Ready | Wait for master API to come back online
```

```suggestion
  - name: OCP | Wait for Node Ready | Wait for master API to come back online
```
I think the task naming convention that is used here is derived from the task filename, so we should leave it saying Node even though it only applies to masters.

---

Reverted change.